CDCgov/prime-reportstream
# Experience/7057 Message Sender Details API

This PR ...

Test Steps:
1. rebuild app
2. log in as an admin/get an admin user's bearer token - only admins have access to this endpoint
3. use the token to make a GET request to /api/message/<covid_results_metadata_id>
*make sure there is a covid_results_metadata_id where the report_id and message_id exist in the action_log table
if there are results, the message details object, warnings array, and errors array are sent back
if there are no results, a message will appear "No message found."

## Changes
- *Include a comprehensive list of changes in this PR*
- *(For web UI changes) Include screenshots/video of changes*

## Checklist

### Testing

### Process

## Linked Issues
- Fixes #7057 

## To Be Done
*Create GitHub issues to track the work remaining, if any*
- #issue

## Specific Security-related subjects a reviewer should pay specific attention to
- Does this PR introduce new endpoints?
    - new endpoint A
    - new endpoint B
- Does this PR include changes in authentication and/or authorization of existing endpoints?
- Does this change introduce new dependencies that need vetting?
- Does this change require changes to our infrastructure?
- Does logging contain sensitive data?
- Does this PR include or remove any sensitive information itself?
